src/index.js

```diff
@@ -11,10 +11,16 @@ const defaultOptions = {
 const mergeImages = (sources = [], options = {}) => new Promise(resolve => {
 	options = Object.assign({}, defaultOptions, options);
 
+	if (options.Canvas && !options.Image && !options.Canvas.Image) {
```

Comment:
I don't think we need to worry about manually throwing here.

When used with canvas people should've already read the docs and know what they need. If they pass in `options.Canvas` and forget `options.Image` they should get an `Image is undefined` error when `new Image()` is called anyway.